packages/amplify_api/lib/method_channel_api.dart

```diff
@@ -143,15 +143,15 @@ class AmplifyAPIMethodChannel extends AmplifyAPI {
   }
 
   Future<GraphQLResponse<T>> _getMethodChannelResponse<T>({
-    @required methodName,
-    @required GraphQLRequest<T> request,
+    required methodName,
+    required GraphQLRequest<T> request,
   }) async {
     try {
       final Map<String, dynamic> result =
-          await _channel.invokeMapMethod<String, dynamic>(
+      (await (_channel.invokeMapMethod<String, dynamic>(
         methodName,
         request.serializeAsMap(),
-      );
+      )))!;
```

Comment:
Are we sure this is non-null?

Reply:
thanks we can't be 100% sure so added the changes we did in the other plugins: 

```

      final Map<String, dynamic>? data = (await (_channel
          .invokeMapMethod<String, dynamic>(methodName, inputsMap)));
      if (data == null) throw nullMapFromMethodChannelException;
```